src/services/ArrayManager.php

```diff
@@ -1,14 +1,14 @@
 <?php
 
-namespace SwitchViews\Services;
+namespace SwitchViews\services;
```

Comment:
It is ideal to that namespace names should start with Capital Letter. In this case **services** should be **Services**